info.csv

```diff
@@ -20,3 +20,4 @@ Ankit Singh,             17SCSE114059,1713114009,            Web Development,2,1
 ketan singh,             17SCSE101558,1713101536,            Web development,2,2,         NA,                          web development,            NA,                  ketansingh343@gmail.com,         https://github.com/KeTan05,              https://www.linkedin.com/in/ketan-singh-a6093816b/,        NA
 Rishabh Arora,           18SCSE1010692,18021011915,           Web Development,1,2,         HTML CSS JavaScript,         Web Development,          Previous School Website, rishabharora.ns@gmail.com,     https://github.com/RishabhAroraa,       https://www.linkedin.com/in/rishabh-arora-b78333156/,       NA
 Monika Sharma,17SCSE101760,1713101727,Android Development,2,1,"Java, C++",Android development AI Cloud,https://github.com/Monika11sharma/HexOpCalci,monikasharma1899@gmail.com,https://github.com/Monika11sharma,https://www.linkedin.com/in/monika-sharma-666b28159/,Gryffindore
+mohd shohib rayeen,17SCSE101517,1713101499,Machine Learning,2,5,python,competitive coding and AI & ML,NA,shoaibm897@gmail.com,https://github.com/Dikisukhi,https://www,linkedin.com/in/mohd-shoaib-5baa46183,NA
```

Comment:
Hey @Dikisukhi,
Check your LinkedIn URL again. You've added a ` ' ` instead of ` . `